libs/positions/src/lib/positions-table.tsx

```diff
@@ -153,9 +153,19 @@ export const PositionsTable = forwardRef<AgGridReact, PositionsTableProps>(
             'color-vega-red': ({ value }: { value: string }) =>
               Number(value) < 0,
           }}
-          valueFormatter={({ value, data }: ValueFormatterParams) =>
```

Comment:
positions-table is totally changed in #473, PR is waiting for review

Reply:
reverted the position table changes so we don't get conflicts while waiting